layouts/shortcodes/discordInvite.html

```diff
@@ -0,0 +1,25 @@
+<!--
+Shortcode for Discord Invite
+Usage:
+< discordInvite invite_code=<> size=<> color=<> image=<> >
+ -->
+
+<!-- #5865F2 is discord's offical "blurple" color -->
+<!-- Large, medium, small breakpoints are arbitrary. Currently, just using tailwind width classes, but this can be tweaked however needed. -->
+<a {{ with .Get "invite_code" }} href="https://discord.gg/{{ . }}" {{ end }} >
+<img src="/img/Discord-Logo+Wordmark-White.png"
```

Comment:
We should probably add this to the theme or pull that image from a cdn... does discord even have a cdn for this sort of thing?

Reply:
I tried just linking to the image on https://discord.com/branding but it appear Discord blocks linking to that that so I think we would have to self host to use it. I mainly used to for filler so whatever we need to change it to is fine.